Segments/Oakvael.mapproj

```diff
@@ -111552,7 +111551,6 @@
         BaseDodge = 14,
         HideDetection = 21,
         Experience = 1755,
-        BasePenetration = ShieldPenetration.VeryLight,
```

Comment:
For Very light removal, you might want to switch that to "none".

Reply:
Got it.